### PR TITLE
dmp 2046 add value to turn off API

### PR DESCRIPTION
**Pull request by chrisdavecm**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DMP-2046

### Change description ###
Added api config to turn off

### Checklist

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)
- [X] Does this PR introduce a breaking change

### CODE REVIEW

File: apps/darts-modernisation/darts-automated-tasks/stg.yaml [hunk 1]
```diff
@@ -13,3 +13,4 @@ spec:
         DARTS_LOG_LEVEL: DEBUG
         RESTART_ME: '007'
         AUTOMATED_TASK_MODE: true
+        API_POD: false
```
Comment (mestebanez) [inline apps/darts-modernisation/darts-automated-tasks/stg.yaml:16]: Maybe way off the mark here so please ignore me but can we not use AUTOMATED_TASK_MODE

Comment (chrisdavecm) [reply apps/darts-modernisation/darts-automated-tasks/stg.yaml:16]: You can here, but the integrated tests don't work as they expect both things to be true

Comment (davet1985) [reply apps/darts-modernisation/darts-automated-tasks/stg.yaml:16]: Can you make this change for all environments?

File: apps/darts-modernisation/darts-automated-tasks/prod.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+metadata:
+  name: darts-automated-tasks
+  namespace: darts-modernisation
+spec:
+  releaseName: darts-automated-tasks
+  values:
+    java:
+      environment:
+        DARTS_GATEWAY_URL: https://darts-gateway.staging.platform.hmcts.net
+        DARTS_PORTAL_URL: https://darts.staging.apps.hmcts.net
```
Comment (davet1985) [inline apps/darts-modernisation/darts-automated-tasks/prod.yaml:12]:
```suggestion
        DARTS_GATEWAY_URL: https://darts-gateway.platform.hmcts.net
        DARTS_PORTAL_URL: https://darts.apps.hmcts.net
```

File: apps/darts-modernisation/darts-automated-tasks/sbox.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+metadata:
+  name: darts-automated-tasks
+  namespace: darts-modernisation
+spec:
+  releaseName: darts-automated-tasks
+  values:
+    java:
+      environment:
+        DARTS_GATEWAY_URL: https://darts-gateway.staging.platform.hmcts.net
+        DARTS_PORTAL_URL: https://darts.staging.apps.hmcts.net
```
Comment (davet1985) [inline apps/darts-modernisation/darts-automated-tasks/sbox.yaml:12]:
```suggestion
        DARTS_GATEWAY_URL: https://darts-gateway.sandbox.platform.hmcts.net
        DARTS_PORTAL_URL: https://darts.sandbox.apps.hmcts.net
```

File: apps/darts-modernisation/darts-automated-tasks/test.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+metadata:
+  name: darts-automated-tasks
+  namespace: darts-modernisation
+spec:
+  releaseName: darts-automated-tasks
+  values:
+    java:
+      environment:
+        DARTS_GATEWAY_URL: https://darts-gateway.staging.platform.hmcts.net
+        DARTS_PORTAL_URL: https://darts.staging.apps.hmcts.net
```
Comment (davet1985) [inline apps/darts-modernisation/darts-automated-tasks/test.yaml:12]:
```suggestion
        DARTS_GATEWAY_URL: https://darts-gateway.test.platform.hmcts.net
        DARTS_PORTAL_URL: https://darts.test.apps.hmcts.net
```

**Review by davet1985 — CHANGES_REQUESTED** [review]
Need to amend the env var values for each env.

**Review by davet1985 — APPROVED** [review]

**Pull request merged**